### DISCUSSION TITLE
Pros and Cons about aptos-core fork repo

### DISCUSSION THREAD

**Discussion by 0xmovses** [movementlabsxyz/movement#118] [General]
Seperate Repo = apto-core fork seperate repo (current strategy)
Monorepo = submodule inside monorepo (proposed strategy)

| **Seperate Repo - Pros**                | **Seperate Repo - Cons**                | **Monorepo - Pros**                | **Monorepo - Cons**                |
|----------------------------------|----------------------------------|----------------------------------|----------------------------------|
| Clear seperation of concerns                  | Can't make quick updates        | Enable fast changes             | Easier to mess up git workflows            | 
| Established flow already, clean rev tag versions        |      | All code in one place               | Adds bloat into the repo      | w
|       |           |             |       Workspace within a Workspace Cargo doesn't like     |

**Comment by l-monninger** [comment]
> Not clear where changes to aptos-core should be made fork repo or submodule

I disagree with this. They're one in the same.

**Comment by l-monninger** [comment]
**Monorepo - Cons**: easier to mess up git workflows.

**Reply by 0xmovses** [reply]
Updated the table, and removed the old point above.
